src/go/rpk/pkg/cli/cmd/wasm/deploy_test.go

```diff
@@ -141,6 +141,9 @@ func TestNewDeployCommand(t *testing.T) {
 						}, {
 							Key:   []byte("sha256"),
 							Value: hashContent[:],
+						}, {
```

Comment:
maybe add a test with an unknown type to make sure the validation passes

Reply:
done